packages/discovery-provider/src/api/v1/cid_data.py

```diff
@@ -18,10 +18,31 @@ def get_data(cid: str):
     return type, metadata
 
 
-CID_METDATA_ROUTE = "/<string:metadata_id>"
+CID_METADATA_ROUTE = "/<string:metadata_id>"
 
+cid_data = full_ns.model(
+    "cid_data",
+    {
+        "collectibles": fields.Raw(required=False),
+        "associated_sol_wallets": fields.Raw(required=False),
```

Comment:
Could the wallet ones be a little stronger typed? Don't love pulling raw objects out in client land.
I get it for collectibles since they get complex really fast.

Reply:
they don't type nicely in flaskrestx here. i can continue to try to figure that out, but they're arbitrary keys. i think that the goal of this work was remove libs, but this is a whole separate refactor in need. just not priority now i think